integration/monai/examples/spleen_ct_segmentation_sim/job/app/config/config_fed_client.json

```diff
@@ -0,0 +1,35 @@
+{
+  "format_version": 2,
+
+  "executors": [
+    {
+      "tasks": [
+        "train", "submit_model", "validate"
+      ],
+      "executor": {
+        "id": "executor",
+        "path": "monai_nvflare.client_algo_executor.ClientAlgoExecutor",
+        "args": {
+          "client_algo_id": "client_algo",
+          "key_metric": "val_mean_dice"
+        }
+      }
+    }
+  ],
+
+  "task_result_filters": [
+  ],
+  "task_data_filters": [
+  ],
+
+  "components": [
+    {
+      "id": "client_algo",
+      "path": "monai.fl.client.MonaiAlgo",
+      "args": {
+        "bundle_root": "config/spleen_ct_segmentation",
```

Comment:
Does this need to change to "spleen_ct_segmentation_sim"?

Reply:
No, bundle_root is relative to the app_root.